docs/resources/Guild.md

```diff
@@ -144,6 +144,9 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 | TICKETED_EVENTS_ENABLED          | guild has enabled ticketed events                                                                                   |
 | MONETIZATION_ENABLED             | guild has enabled monetization                                                                                      |
 | MORE_STICKERS                    | guild has increased custom sticker slots                                                                            |
+| THREE_DAY_THREAD_ARCHIVE         | guild has access to the three day archive time for threads                                                          |
+| SEVEN_DAY_THREAD_ARCHIVE         | guild has access to the seven day archive time for threads                                                          |
+| PRIVATE_THREADS                  | guild has access to create private threads                                                                          |
```

Comment:
Which PremiumTier is needed for `PRIVATE_THREADS`

Reply:
2

Reply:
Thank you <3